package.json

```diff
@@ -23,7 +23,11 @@
     "docs:build": "npm run docs:prepare && gitbook build -g redux-saga/redux-saga",
     "docs:watch": "npm run docs:prepare && gitbook serve",
     "docs:publish": "npm run docs:clean && npm run docs:build && cp CNAME _book && touch _book/.nojekyll && gh-pages -d _book -r git@github.com:redux-saga/redux-saga.git -m 'update book' -t",
-    "bundlesize": "lerna run --parallel bundlesize"
+    "bundlesize": "lerna run --parallel bundlesize",
+    "changeset": "changeset",
```

Comment:
is there a way to add changesets (and automate release process) for beta releases etc?

Reply:
We're working on prereleases at the moment:

https://github.com/atlassian/changesets/pull/183

Reply:
❤️ great!